### PR TITLE
[14.0][MIG] partner_pricelist_search: Migration to 14.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 14.0.

Please @carlosdauden and @joao-p-marques can you review it?

@Tecnativa TT31807

### CODE REVIEW

File: partner_pricelist_search/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+class Pricelist(models.Model):
+    _inherit = "res.partner"
+
+    property_product_pricelist = fields.Many2one(
+        search="_search_property_product_pricelist"
+    )
+
+    @api.model
+    def _search_property_product_pricelist(self, operator, value):
+        if operator == "=":
```
Comment (pedrobaeza) [inline partner_pricelist_search/models/res_partner.py:17]: I think you can use the same technique as used in other module. @CarlosRoca13 can you point to the module with the operator import?

File: partner_pricelist_search/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Carlos Dauden
```
Comment (pedrobaeza) [inline partner_pricelist_search/readme/CONTRIBUTORS.rst:3]: Put yourself here

File: partner_pricelist_search/models/res_partner.py (hunk 1 above)
Comment (CarlosRoca13) [reply partner_pricelist_search/models/res_partner.py:17]: Yes, sure, here you have the PR were was added that logic https://github.com/OCA/stock-logistics-warehouse/pull/1246

Comment (pedrobaeza) [reply partner_pricelist_search/models/res_partner.py:17]: After seeing it, it's not as easy and this code is enough.

**Review by pedrobaeza — APPROVED** [review]

**Review by joao-p-marques — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**